app/web/src/features/communities/events/EventForm.tsx

```diff
@@ -127,8 +138,14 @@ export default function EventForm({
     register,
     setValue,
     watch,
+    formState: { touched },
   } = useForm<CreateEventData>();
 
+  // This is a bit of a dirty workaround, but I have no ideas why react-hook-form doesn't pick up when
+  // the datepicker field has been touched...
+  const isStartDateTouched = useRef(false);
+  const isEndDateTouched = useRef(false);
```

Comment:
This is the part I'm not happy about doing, because changing the date using the datepicker doesn't seem to get detected by react-hook-form as a "touched" field and so without this, updating the event time becomes impossible unless you touch the startTime/endTime fields as well, as the conditional logic checks either the date or time has been touched to decide whether to send that in the `UpdateEventReq` to the backend.